keyboards/mariorion_v25/readme.md

```diff
@@ -0,0 +1,30 @@
+# MariOrion v2.5
+
+![mariorion_v25_1](https://i.imgur.com/3OrUK2ph.jpeg)
+![mariorion_v25_2](https://i.imgur.com/OvMBHOlh.jpeg)
+
+*A short description of the keyboard/project*
```

Comment:
Could a short description be added please?

Be a good place for informing users how to differentiate between building/flashing `prod` and `proto` versions

Reply:
Sure can do that :) lemme come up with smth to write haha~

Reply:
Is this update explanatory enough? If it's ok lemme know if not lemme know what to add :D